### ISSUE TITLE
eslint error when importing module "import/no-unresolved"

### ISSUE THREAD

**Issue by jzilg** [marcofugaro/browserslist-to-esbuild#1]
Hello,

the module works fine but when importing in webpack.config.js I get a eslint error

`error  Unable to resolve path to module 'browserslist-to-esbuild'  import/no-unresolved`

Best

**Comment by marcofugaro** [comment]
are you doing 
```js
const browserslistToEsbuild = require('browserslist-to-esbuild')
```
and are you sure you installed the module beforehand?

Maybe try to close and reopen your editor so it checks if the dependency is correctly installed.

**Comment by jzilg** [comment]
Yes but I get the same error if I do

`import browserslistToEsbuild from 'browserslist-to-esbuild'`

Yes I installed it and find it in my `node_modules` folder. I restarted my IDE already but I get the same error if I run eslint on the Terminal.
The webpack build works. So the import works and it is only a eslint issue.

If I do 

`const browserslistToEsbuild = require('browserslist-to-esbuild/src/index')`

I do not get the eslint error but then the build fails.

**Comment by marcofugaro** [comment]
What node version are you using?

**Comment by jzilg** [comment]
node v14.16.0
npm 7.18.1

**Comment by marcofugaro** [comment]
Pushed a fix, you can test with `npm install --save-dev browserslist-to-esbuild@marcofugaro/browserslist-to-esbuild`.

I'll publish a new version as soon as npm comes back online: https://status.npmjs.org/

**Comment by jzilg** [comment]
Yes that works. Thank you very much! :)

**Comment by marcofugaro** [comment]
Fixed in new version `v1.1.1` 🙂
